omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -58,6 +59,28 @@ def solr_root
     host
   end
 
+  def elastic_search_major_version
+    default_version = 2
+    if node['private_chef']['opscode-solr4']['external']
+      elastic_search_url = node['private_chef']['opscode-solr4']['external_url']
+      begin
+        elastic_search_uri = URI.parse(elastic_search_url)
+        req = Net::HTTP::Get.new(elastic_search_uri)
+        res = Net::HTTP.start(elastic_search_uri.hostname, elastic_search_uri.port, use_ssl: elastic_search_uri.scheme == "https") do |http|
+          http.request(req)
+        end
+        version = JSON.parse(res.body)['version']['number'].split('.').first.to_i
+        version = default_version if version == 0
+        version
```

Comment:
I considered that - but then I didn't know what we did support.  There's only ES 1, 2 and 5 right now (6 is on the way).  I guess we don't support 1.  Are there any minor versions in 2 that we don't support?